packages/ui/src/utils/renderWithTheme.ts

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render, RenderOptions, RenderResult } from '@testing-library/react';
+
+import { ThemeProvider } from '../theme';
+
+export const renderWithTheme = (ui: ReactElement, options: Omit<RenderOptions, 'queries'> = {}): RenderResult => {
```

Comment:
🤔 This function should have the same interface the default `render` has. Here you omit the `queries` property and disallow to provide additional wrappers (they would overwrite the `ThemeProvider` if passed)